docs/qa.md

```diff
@@ -0,0 +1,25 @@
+# List of paths through Figgy for QA
+
+* CRUD (Create/Read/Update/Delete)
+  * Scanned Resource
+    * With/without imported metadata
+  * Scanned Map
+  * Raster Resource
+  * Vector Resource
+  * Numismatics
```

Comment:
suggestion: Maybe specifically mention issue and coin form select boxes.

Reply:
we should also add CDL!